### PR TITLE
User can edit article

### PR DESCRIPTION
**Pull request by jonas-masharqa**
[PT-Link](https://www.pivotaltracker.com/story/show/169253793)

## PR Content
```
As a user,
In order to change articles I have already written
I would like to be able to edit articles
```

## PR Content
- Integration test Happy & Sad path
- Fixture
- EditFormInput component
- PUT request function in module

### CODE REVIEW

File: src/Components/SingleArticle.js [hunk 1]
```diff
@@ -18,24 +24,79 @@ class SingleArticle extends Component {
...
+  submitChangeHandler = async () => {
+    const { title, content, image } = this.state
+    let response = await editArticle(title, content, image)
+
+    if (response.status === 200) {
+      this.setState({
+        responseMessage: response.data.message
+      })
+    } else {
+      debugger;
```
Comment (AlexSaxena) [inline src/Components/SingleArticle.js]: Please remove Debugger here.

File: src/Modules/ArticlesData.js [hunk 2]
```diff
@@ -34,8 +34,24 @@ const getSpecificArticle = async (articleId) => {
     let response = await axios.get(apiUrl + `articles/${articleId}`)
     return response
   } catch(error) {
+    debugger;
```
Comment (AlexSaxena) [inline src/Modules/ArticlesData.js]: Please remove this debugger

**Review by lucamarial — APPROVED** [review]

File: src/Components/SingleArticle.js (hunk 1 above)
Comment (clalil) [reply src/Components/SingleArticle.js]: Changed accordingly.

File: src/Modules/ArticlesData.js (hunk 2 above)
Comment (clalil) [reply src/Modules/ArticlesData.js]: Done!

File: src/Modules/ArticlesData.js [hunk 3]
```diff
@@ -38,4 +38,19 @@ const getSpecificArticle = async (articleId) => {
...
+const editArticle = async (title, content, image) => {
+  try {
+    let response = await axios.put(apiUrl + 'articles',
+    {
+      title: title,
+      content: content,
+      image: image
+    })
+    return response
+
```
Comment (lucamarial) [inline src/Modules/ArticlesData.js]: remove blank line

**Review by lucamarial — APPROVED** [review]

**Review by clalil — APPROVED** [review]

**Review by lucamarial — APPROVED** [review]

**Review by AlexSaxena — APPROVED** [review]

**Review by shsteindorsson — APPROVED** [review]

**Review by oliverochman — APPROVED** [review]

**Pull request merged**